service/controllers/contactsController.js

```diff
@@ -1,11 +1,11 @@
-const Contact = require("./Schemas/contactSchema");
+const { Contact } = require("../Schemas/contactSchema");
 
 const listContacts = async () => {
   try {
     const contacts = await Contact.find();
```

Comment:
```suggestion
    const contacts = await Contact.find({owner});
```
Razem z autoryzacją trzeba się upewnić że tylko osoba która jest twórcą danego kontaktu może go edytować. I to we wszystkich ścieżkach które obsługują kontakty musisz dodać obsługę tego kontaktu tylko przez autoryzowanego użytkownika. Czyli są dwa poziomy autoryzacji. Pierwszy poziom autoryzacji umożliwia dostęp do ścieżki a drugi poziom autoryzacji umożliwia edycję rekordów w bazie danych. Ten drugi poziom autoryzacji uzyskujemy poprzez klucz owner.